frontend/pages/_slug/verificaciones/dns/_id/estadisticas.vue

```diff
@@ -102,7 +102,7 @@ export default {
         getData() {
             this.clearData();
 
-            return this.$axios.$get(`${this.getTenant.slug}/verifications/dns-verifications/${this.idParam}/get-data?start_date=${this.dateFilteredStart}&end_date=${this.dateFilteredEnd}`)
+            return this.$axios.$get(`${this.getTenant.slug}/verifications/dns-verifications/${this.idParam}/get-data?start_date=${this.dateFilteredStart || 0}&end_date=${this.dateFilteredEnd || 0}`)
```

Comment:
@MatiasMinchaca why you did this change?

Reply:
The default values in these variables are to avoid an error when making a request to the API since a null value is sent, and response error 500